examples/azure/terraform-azure-nsg-example/main.tf

```diff
@@ -0,0 +1,139 @@
+provider "azurerm" {
```

Comment:
NIT: Add a block at the top that points to the corresponding test file similar to https://github.com/gruntwork-io/terratest/blob/master/examples/azure/terraform-azure-availabilityset-example/main.tf#L1

Reply:
Added comment block as in the availability set example.